### ISSUE TITLE
Error When Tags are empty

### ISSUE THREAD

**Issue by fdmsantos** [Redevaerk/terraform-azurerm-bot-service#4]
# Debug Output/Panic Output

```shell
╷
│ Error: Invalid index
│ 
│   on .terraform/modules/bot.private_endpoint/modules/private-dns-zone/locals-tags.tf line 11, in locals:
│   11:     for key in chunklist(keys(local.merged_tags), 14)[0] : key => lookup(local.merged_tags, key)
│     ├────────────────
│     │ local.merged_tags is empty map of string
│ 
│ The given key does not identify an element in this collection value: the collection has no elements.
╵
```

**Comment by fdmsantos** [comment]
Related with [2](https://github.com/claranet/terraform-azurerm-private-endpoint/issues/2)

**Comment by fdmsantos** [comment]
Fixed in [v1.4.0](https://github.com/Redevaerk/terraform-azurerm-bot-service/releases/tag/v1.4.0)
